digitaltwin_client/src/dt_client_core.c

```diff
@@ -799,6 +799,58 @@ static DIGITALTWIN_CLIENT_RESULT SendInterfacesToRegisterMessage(DT_CLIENT_CORE*
     return result;
 }
 
+// VerifyInterfaceInstancesUnique makes sure that the interfaceInstanceName of each handle is unique.  E.G. one connection cannot have
+// to instances both named "frontCamera".  We check here, instead of solely relying on server policy, because if this fails on server
+// side for MQTT then the connection will be dropped and it will be hard for application developer to debug.
```

Comment:
>the connection will be dropped [](start = 22, length = 30)

Is it worth documenting the error handling protocol from service here? If it is, I recommend changing verbiage from WILL to MAY?